.circleci/common.sh

```diff
@@ -0,0 +1,2 @@
+MINICONDA_VER=4.5.4
```

Comment:
@hepcat72 can you please remove those changes.

Reply:
Whoops.  Sorry about that. Unintended commit.